MANIFEST.in

```diff
@@ -1,2 +1,50 @@
-include README.rst
-include requirements.txt
+include .git-pre-commit
+include LICENSE
+include MANIFEST.in
+include Makefile
+include README.rst
+include fulltext/__init__.py
+include fulltext/__main__.py
+include fulltext/backends/__bin.py

```

Comment:
FYI, You can use a wildcard in `MANIFEST.in`. This may be useful for adding backends... This may allow you to obsolete `generate_manifest`...

```
include fulltext/backends/__*.py
```

https://docs.python.org/2/distutils/sourcedist.html#principle

Reply:
This decision comes from `psutil`. In psutil I've been bitten a couple of times by the fact that the `tar.gz` distribution uploaded on PYPI was missing some (new or moved) files because I forgot to update MANIFEST.in, therefore I ended up auto generating MANIFEST.in like this.
So `generate_manifest` basically is my strategy to forget about updating `MANIFEST.in` in case the directory tree changes.
IMO I think it's worth it.